common/decklist.cpp

```diff
@@ -466,29 +468,74 @@ bool DeckList::saveToFile_Native(QIODevice *device)
 bool DeckList::loadFromStream_Plain(QTextStream &in)
 {
     cleanList();
+    bool inSideboard = false, titleFound = false, isSideboard;
+    int okRows = 0, blankLines = 0;
```

Comment:
Its best to keep the usage of variables as close to their usage as possible. 
The bools and `okRows` can be moved to line `517`, `blankLines` to `505`